config/environments/development.rb

```diff
@@ -37,14 +37,14 @@
   # config.action_view.raise_on_missing_translations = true
 
   # Gives paperclip access to ImageMagick
-  
+
   Paperclip.options[:command_path] = "/usr/local/bin/"
   config.paperclip_defaults = {
     :storage => :s3,
     :s3_credentials => {
-      :bucket => ENV['S3_BUCKET_NAME'],
-      :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
-      :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
+      :bucket => ENV['S3_DEV_BUCKET_NAME'],
+      :access_key_id => ENV['AWS_DEV_ACCESS_KEY_ID'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.